### PR TITLE
Update release notes draft to version v1.28.0-alpha.2

### PR DESCRIPTION
**Pull request by fsmunoz**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.28.0-alpha.2

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

**Comment by sanchita-07** [comment]
cc @ramrodo

**Comment by ramrodo** [comment]
Great!
/lgtm

File: releases/release-1.28/release-notes/maps/pr-116685-map.yaml
Comment (AnaMMedina21) [inline releases/release-1.28/release-notes/maps/pr-116685-map.yaml:4]:
```suggestion
    `kube-apiserver` will now always remove its endpoint from Kubernetes service during
```

File: releases/release-1.28/release-notes/maps/pr-117120-map.yaml
Comment (AnaMMedina21) [inline releases/release-1.28/release-notes/maps/pr-117120-map.yaml:3]:
```suggestion
  text: Bumped metrics-server to `v0.6.3`.
```

File: releases/release-1.28/release-notes/maps/pr-117138-map.yaml
Comment (AnaMMedina21) [inline releases/release-1.28/release-notes/maps/pr-117138-map.yaml:3]:
```suggestion
  text: Added '--concurrent-job-syncs' flag for `kube-controller-manager` to set the
```

File: releases/release-1.28/release-notes/maps/pr-117211-map.yaml
Comment (AnaMMedina21) [inline releases/release-1.28/release-notes/maps/pr-117211-map.yaml:5]:
```suggestion
    that allow users to monitor requests to authorization webhooks, split by the results.
```

File: releases/release-1.28/release-notes/maps/pr-117655-map.yaml
Comment (AnaMMedina21) [inline releases/release-1.28/release-notes/maps/pr-117655-map.yaml:3]:
```suggestion
  text: Removed (GA) feature gate for: `DelegateFSGroupToCSIDriver`.
```

File: releases/release-1.28/release-notes/maps/pr-117660-map.yaml
Comment (AnaMMedina21) [inline releases/release-1.28/release-notes/maps/pr-117660-map.yaml:4]:
```suggestion
    `kubeadm`: introduced a new feature gate `UpgradeAddonsBeforeControlPlane` to
```

File: releases/release-1.28/release-notes/maps/pr-117656-map.yaml
Comment (AnaMMedina21) [inline releases/release-1.28/release-notes/maps/pr-117656-map.yaml:3]:
```suggestion
  text: Removed (GA) feature gate for: `DevicePlugins`.
```

File: releases/release-1.28/release-notes/maps/pr-118029-map.yaml
Comment (AnaMMedina21) [inline releases/release-1.28/release-notes/maps/pr-118029-map.yaml:3]:
```suggestion
  text: Code blocks in `kubectl {$COMMAND}--help` will move right by 3 indentation.
```

File: releases/release-1.28/release-notes/maps/pr-118270-map.yaml
Comment (AnaMMedina21) [inline releases/release-1.28/release-notes/maps/pr-118270-map.yaml:3]:
```suggestion
  text: '''`kubeadm`: will now throw warnings instead of errors for deprecated feature gates.'''
```

**Review by AnaMMedina21 — APPROVED** [review]
looks great, just left a few suggestions :)

File: releases/release-1.28/release-notes/maps/pr-117120-map.yaml
Comment (fsmunoz) [reply releases/release-1.28/release-notes/maps/pr-117120-map.yaml:3]: The Docs [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) do not use backticks for version numbers, should we use them in release notes? No problem in changing it, but would like more input on it.

File: releases/release-1.28/release-notes/maps/pr-117301-map.yaml
Comment (fsmunoz) [reply releases/release-1.28/release-notes/maps/pr-117301-map.yaml:3]: Same as above about version formats.

File: releases/release-1.28/release-notes/maps/pr-117656-map.yaml
Comment (fsmunoz) [reply releases/release-1.28/release-notes/maps/pr-117656-map.yaml:3]: I also disliked the "GAed" expression, but it's the one [used in the official documentation](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates-removed/) (although with an apostrophe).

What do you think about something like:

"Removed GA'ed feature gate: "

**Review by sanchita-07 — APPROVED** [review]

File: releases/release-1.28/release-notes/maps/pr-117656-map.yaml
Comment (AnaMMedina21) [reply releases/release-1.28/release-notes/maps/pr-117656-map.yaml:3]: that would work! I like it, I hadn't checked how the docs had it

File: releases/release-1.28/release-notes/maps/pr-117120-map.yaml
Comment (AnaMMedina21) [reply releases/release-1.28/release-notes/maps/pr-117120-map.yaml]: Woots! And for context, I think I learned the backticks from @ramrodo. I think they read nicer on Release Notes with them, but maybe we should document it somewhere for the release notes team?

File: releases/release-1.28/release-notes/maps/pr-117656-map.yaml
Comment (fsmunoz) [reply releases/release-1.28/release-notes/maps/pr-117656-map.yaml]: Done, added changes and you as co-author the commit.

File: releases/release-1.28/release-notes/maps/pr-117656-map.yaml
Comment (fsmunoz) [reply releases/release-1.28/release-notes/maps/pr-117656-map.yaml:1]: Fixed in last commit.

File: releases/release-1.28/release-notes/maps/pr-117211-map.yaml
Comment (fsmunoz) [reply releases/release-1.28/release-notes/maps/pr-117211-map.yaml:5]: Not sure about this one, in the code the help message says this:
```
Counter of authorization attempts broken down by result. It can be either 'allowed', 'denied', 'no-opinion' or 'error'.
```

I've almost always seen "split by result" used (in e.g. analytics packages), I think it fits well with the itendend meaning: the user will get the requests split by the result, which is a predefined value. Open to debate it further though!

**Comment by fsmunoz** [comment]
@AnaMMedina21 @sanchita-07 I've added all changes and rebased to clean the commit history before merge.

**Comment by fsmunoz** [comment]
@ramrodo lgtm label was removed due to changes, and while I can't promise there won't be more, we seem to be very close to the final result :)

**Review by sanchita-07 — APPROVED** [review]
/lgtm

**Comment by sanchita-07** [comment]
Thank you!

**Comment by sanchita-07** [comment]
cc @kubernetes/release-team-leads 
Needs approval for merging the PR.

**Pull request merged**
